components/vf-button/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
No need to bump the version here. When we do our release, `lerna` will automatically update this.